docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -19,7 +19,11 @@ yarn add aws-amplify @aws-amplify/ui-react
 </TabItem>
 </Tabs>
 
-Wrap your application in `<AmplifyProvider/>` and include default styling:
+After adding the `aws-amplify` and `@aws-amplify/ui-react` dependencies you are now ready to add any of our [components](/components) to your application.
+
+### Theming
+
+To add [theming](/theming) wrap your application in the `AmplifyProvider` component and include the default styling.
 
 <Tabs>
 <TabItem title="Create React App">
```

Comment:
I can't comment on the existing code, but it seem weird to name the function App and then have an `<App />` component inside. Maybe put a `<View>` component with "Your app here" string inside.

Reply:
OK I'll change it to `View`